tests/integration/nupic/opf/hotgym_regression_test.py

```diff
@@ -0,0 +1,77 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2015, Numenta, Inc.  Unless you have purchased from
+# Numenta, Inc. a separate commercial license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+"""Regression test that checks for differences in hotgym results.
+
+If the prediction results differ then the test fails and must be
+explicitly updated to match the new results.
+"""
+
+import collections
+import csv
+import os
+import pkg_resources
+import shutil
+import unittest
+
+from nupic.frameworks.opf import experiment_runner
+
+
+
+class HotgymRegressionTest(unittest.TestCase):
+  """Hotgym regression test to validate that predictions don't change."""
+
+
+  def testHotgymRegression(self):
+    experimentDir = pkg_resources.resource_filename(
+        "nupic",
+        os.path.join("..", "examples", "opf", "experiments", "multistep",
+                     "hotgym")
+    )
+
+    resultsDir = os.path.join(experimentDir, "inference")
```

Comment:
I'm not sure there is any way around this with the current experiment runner. Note that this path I construct is something created by the experiment runner. I create the path just so I can read the results and clean them up afterwards.